src/decisionengine/framework/taskmanager/TaskManager.py

```diff
@@ -111,11 +113,19 @@ def __init__(self, channel_dict, channel_name):
 
         delogger.debug("Creating channel source")
         self.sources = _make_workers_for(channel_dict["sources"], Source, channel_name)
-        delogger.debug("Creating channel logicengine")
-        self.le_s = _make_workers_for(channel_dict["logicengines"], LogicEngine, channel_name)
+
         delogger.debug("Creating channel publisher")
         self.publishers = _make_workers_for(channel_dict["publishers"], Publisher, channel_name)
 
+        delogger.debug("Creating channel logicengine")
+        configured_le_s = channel_dict.get("logicengines")
+        if configured_le_s is None:
+            configured_le_s = passthrough_configuration(channel_dict["publishers"].keys())
+        if len(configured_le_s) > 1:
+            raise RuntimeError("Cannot support more than one logic engine per channel.")
```

Comment:
Should we have a test that hits this exception?

Reply:
Yep.  Pushed.